### DISCUSSION TITLE
Nick Pakistan

### DISCUSSION THREAD

**Discussion by spongebob849** [iptv-org/community#762] [Q&A]
### Channel Name

Nick Pakistan

### Channel ID

Nickelodeon.pk

### Country

Pakistan

### Language

English

### Website

http://www.nickpakistan.tv

### Notes

https://en.wikipedia.org/wiki/Nickelodeon_(Pakistani_TV_channel)

**Comment by freearhey** [comment]
According to our database this channel was closed 28 October, 2024.

Source: https://iptv-org.github.io/channels/pk/Nickelodeon
